### DISCUSSION TITLE
Add lastUpdated to currentPriceLevel

### DISCUSSION THREAD

**Discussion by Danielhiversen** [3lbits/API-nettleie-for-styring#9] [Ideas]
Regarding currentPriceLevel in 
https://github.com/DIGINenergi/API-nettleie-for-styring/blob/793d2f90b18d9d199127d499661ddb4496dad030/doc/DiginGridTariffAPI.v0_8_meteringpointsgridtariffs_json_output_dailymax_example1.json#L130

Would it be useful to add a date field indicating when it is last updated?
Since the level might change the next time the grid company receives consumption data from the AMS meter, and the API user does not know when the grid company last got consumption data and updated the level.

```
  "currentPriceLevel": {
    "currentLevelId": 3,
    "currentLevelValue": null,
    "lastUpdated": "2021-09-17T01:00:00+02:00"
  }
```

**Comment by areroshaa** [comment]
@Danielhiversen this is a good input. Do you mean the timestamp should be the latest consumption data the grid company received from the meter for this MPID?

**Reply by Danielhiversen** [reply]
Yes,  the latest consumption data the grid company received and used for calculating the price level?

**Reply by areroshaa** [reply]
v1.0.2 of the schema now contains:
  "MeteringPointDetails": {
    "description": "Definition of meteringPointId and levelValue/lastUpdated for fixed price level",
    "type": "object",
    "properties": {
      "meteringPointId": {
        "description": "meteringPointId. Ex. 707057500000000001",
        "type": "string"
      },
      "levelValue": {
        "description": "Value of the max hour setting the level of fixedprice. NOT IN USE YET AS IT REQUIRES A HIGHER LEVEL OF AUTHENTICATION AND AUTHORIZATION. Ex. 9.0000",
        "type": "number",
        "format": "double",
        "nullable": true
      },
      "lastUpdated": {
        "description": "The end time(hour) of the last calculated hour consumption(based on meter reading or estimation) this month. Null if FixedPriceConfiguration.basis = fixed. Ex. 2021-09-17T02:00:00+02:00",
        "type": "string",
        "format": "date-time",
        "nullable": true
      }
    },
    "additionalProperties": false
